html/service.php

```diff
@@ -0,0 +1,33 @@
+<!-- Service Start -->
+<div class="container-fluid service position-relative px-5 mt-5" style="margin-bottom: 135px;">
+  <div class="container">
+    <div class="row g-5">
+      <div class="col-lg-4 col-md-6">
+        <div class="bg-primary border-inner text-center text-white p-5">
+          <h4 class="text-uppercase mb-3"><?=TXT_BIRTHDAY2;?> Cake</h4>
+          <p><?=TXT_PARAGRAPH_BIRTHDAY;?></p>
+          <a class="text-uppercase text-dark" href=""> <?BUTTON_READ_MORE;?> <i class="bi bi-arrow-right"></i></a>
+        </div>
+      </div>
+      <div class="col-lg-4 col-md-6">
+        <div class="bg-primary border-inner text-center text-white p-5">
+          <h4 class="text-uppercase mb-3"><?=TXT_WEDDING2;?> Cake</h4>
```

Comment:
```suggestion
          <h4 class="text-uppercase mb-3"><?=TXT_BIRTHDAY2;?></h4>
```
Agrega la palabra Cake en la constante